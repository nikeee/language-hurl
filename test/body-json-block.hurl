POST https://example.org/InStock
```json
{
  "a": 1
}
```
HTTP 200
