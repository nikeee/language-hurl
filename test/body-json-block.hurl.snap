>POST https://example.org/InStock
#^^^^ source.hurl keyword.control.method.hurl
#    ^ source.hurl
#     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl string.quoted.url.hurl string.quoted.url.hurl
>```json
#^^^^^^^ source.hurl meta.embedded.block.json.hurl
>{
#^^ source.hurl meta.embedded.block.json.hurl text.json
>  "a": 1
#^^^^^^^^^ source.hurl meta.embedded.block.json.hurl text.json
>}
#^^ source.hurl meta.embedded.block.json.hurl text.json
>```
#^^^ source.hurl meta.embedded.block.json.hurl
>HTTP 200
#^^^^ source.hurl constant.language.version.hurl
#    ^^^^ source.hurl constant.numeric.status.hurl
>